src/ui/components/Badge/styles.scss

```diff
@@ -0,0 +1,35 @@
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+@mixin icon($name) {
```

Comment:
This is pretty cool. We should use it in [`Icon/styles.scss`](https://github.com/mozilla/addons-frontend/blob/master/src/ui/components/Icon/styles.scss) :) but if that's too big of a change for this patch then maybe just a file an issue to fix it up later.